src/wing/CCPACSWingCell.cpp

```diff
@@ -189,7 +188,7 @@ bool CCPACSWingCell::IsInside(double eta, double xsi) const
     }
 }
 
-void CCPACSWingCell::ReadCPACS(TixiDocumentHandle tixiHandle, const std::string &cellXPath)
+void CCPACSWingCell::ReadCPACS(const TixiDocumentHandle& tixiHandle, const std::string &cellXPath)
```

Comment:
I just had a discussion with a colleague of mine (https://github.com/MFHava) who argued that you can also take the handle by value. Many handle implementations design their handles as value types identifying shared resources. However, if the resource's lifetime should be managed and end with the destruction of the last handle, it has to be reference counted. And a reference counted handle can always be constructed by implementing a single owernship class representing the resouce and wrapping e.g. a std::shared_ptr<T> around it. The shared_ptr can than be freely copied (handled as a value type) and the resource is destroyed with the last shared_ptr's dtor.

The bottom line is, both implementations are valid. I would still prefer the const T& and my colleague thinks it is the less troublesome, if the definition of the handle is changed.